esmvaltool/_config.py

```diff
@@ -142,10 +146,14 @@ def get_project_config(project):
     return CFG[project]
 
 
-def get_institutes(dataset):
+def get_institutes(variable):
     """Return the institutes given the dataset name in CMIP5."""
+    dataset = variable['dataset']
+    project = variable['project']
     logger.debug("Retrieving institutes for dataset %s", dataset)
-    return CFG['CMIP5']['institutes'].get(dataset, [])
+    if 'institutes' in CFG[project]:
```

Comment:
Maybe replace these three lines by `return CFG.get(project, {}).get(dataset, [])` so the return type is consistently a `list`?

Reply:
Good idea